src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/AutoFlushingNerdbankStream.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    // Intended for use with a party from a Nerdbank FullDuplexStream:
+    // https://github.com/AArnott/Nerdbank.Streams/blob/master/doc/FullDuplexStream.md
+    internal class AutoFlushingNerdbankStream : Stream
```

Comment:
Renamed `AutoFlushingStream` to `AutoFlushingNerdbankStream` to clearly indicate some custom logic specifically to do with `Nerdbank.Streams`. This `Autoflushing` class shouldn't be used with other types of streams.